server/src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Inject, Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { UserServiceImpl } from '../user/user.service.impl';
+import { ValidationServiceImpl } from '../../domain/user/validation.service.impl';
+import { User } from '../../domain/user/user.entity';
+import { Token } from './token.entity';
+import { SignUpResponseDto } from './dto/sign-up-response.dto';
+import { SignInResponseDto } from './dto/sign-in-response.dto';
+
+@Injectable()
+export class AuthService {
+    constructor(
+        @Inject('JwtService') private readonly jwtService: JwtService,
+        @Inject('UserService') private readonly userService: UserServiceImpl,
+        @Inject('ValidationService') private readonly validationService: ValidationServiceImpl,
+    ) {}
+
+    createToken(address: string): Token {
+        if (address === undefined) {
+            throw new Error('address should be defined.');
+        }
+
+        return {
+            expiresIn: 3600,
```

Comment:
i think set config using config file is better

Reply:
@inDlife After discussing it with other reviewers, removed the config file and changed it to hard-coded style.